CITATION.rst

```diff
@@ -1 +0,0 @@
-sunpy/CITATION.rst
```

Comment:
Ideally this file should not show up on the files changed tab of github. You will want to check out this file from the main sunpy repository.